Emgu.CV/Videoio/ICapture.cs

```diff
@@ -1,7 +1,7 @@
 //----------------------------------------------------------------------------
 //  Copyright (C) 2004-2019 by EMGU Corporation. All rights reserved.       
 //----------------------------------------------------------------------------
-#if !(__ANDROID__ || __UNIFIED__ || NETFX_CORE || UNITY_WSA || NETSTANDARD1_4 || UNITY_ANDROID || UNITY_IOS || UNITY_EDITOR || UNITY_STANDALONE || NETCOREAPP1_1)
+#if !(__ANDROID__ || __UNIFIED__ || NETFX_CORE || UNITY_WSA || NETSTANDARD1_4 || UNITY_ANDROID || UNITY_IOS || UNITY_EDITOR || UNITY_STANDALONE || NETCOREAPP1_1 || NETCOREAPP)
```

Comment:
The items affected by this are available for .NET Standard/.NET Core and are needed to use a WCF Client. The types in the code base that need usage removed are mentioned in [this](https://github.com/emgucv/emgucv/issues/1#issuecomment-525006712) comment. I believe all the attributes this change removes are in the nuget package System.ServiceModel.Primitives.

Reply:
Thanks Matt,

Added the reference to .Net Core 3.0.
Looks like XmlSerializerFormatAttribute is [not available](https://apisof.net/catalog/System.ServiceModel.XmlSerializerFormatAttribute) before .Net Standard 1.6. Not sure if the maintainers want to target it as well.

Reply:
The oldest .NET Core version supported is .NET Core 2.1 so all currently supported version of .NET Core support at least .NET Standard 2.0